R/a-helper.r

```diff
@@ -0,0 +1,2 @@
+# Evaluates all arguments (see #81)
+force_all <- function(...) list(...)
```

Comment:
Done, I had to use `@include utils.r` where needed to make sure the Collate will be well done.